Fix libcap compile error:

../../aarch64-wrs-linux-gnu/bin/ld: warning: libattr.so.1, needed by /buildarea/kkang/build-new-aarch64/bitbake_build/tmp/work/aarch64-wrs-linux/libcap/2.22-r5/libcap-2.22/libcap/libcap.so, not found (try using -rpath or -rpath-link)

diff -Nur libcap-2.22/pam_cap/Makefile libcap-2.22.new/pam_cap/Makefile
--- libcap-2.22/pam_cap/Makefile	2010-08-09 08:08:01.000000000 +0800
+++ libcap-2.22.new/pam_cap/Makefile	2013-11-21 10:04:53.763003937 +0800
@@ -7,7 +7,7 @@
 # that this next line does *not* require -lpam on it.) If you think it
 # does, *verify that it does*, and if you observe that it fails as
 # written (and you know why it fails), email me and explain why. Thanks!
-LDLIBS += -L../libcap -lcap
+LDLIBS += -L../libcap -lcap -lattr
 
 all: pam_cap.so
 	$(MAKE) testcompile
diff -Nur libcap-2.22/progs/Makefile libcap-2.22.new/progs/Makefile
--- libcap-2.22/progs/Makefile	2010-08-09 07:56:40.000000000 +0800
+++ libcap-2.22.new/progs/Makefile	2013-11-21 10:05:09.692003874 +0800
@@ -14,7 +14,7 @@
 ifneq ($(DYNAMIC),yes)
 LDFLAGS += --static
 endif
-LDLIBS += -L../libcap -lcap
+LDLIBS += -L../libcap -lcap -lattr
 
 all: $(BUILD)
 
